plugins/completion/coq-thirdparty.nix

```
@@ -34,21 +34,24 @@ in {
           };
         };
       });
-      description = "List of sources";
+      description = ''
+        List of sources. Each source is a free form type, so additional settings like `accept_key`
+        may be specified even if they are not declared by nixvim
```

```suggestion
        List of sources.
        Each source is a free-form type, so additional settings like `accept_key` may be specified even if they are not declared by nixvim.
```

Hopefully I said that I would stop nitpicking :')